code/datums/status_effects/buffs.dm

```diff
@@ -219,13 +219,15 @@
 	owner.adjustBruteLoss(-25)
 	owner.adjustFireLoss(-25)
 	owner.remove_CC()
-	owner.bodytemperature = BODYTEMP_NORMAL
 	if(ishuman(owner))
 		var/mob/living/carbon/human/H = owner
+		H.bodytemperature = H.dna.species.heat_level_1
```

Comment:
Might need `body_temperature` instead. `heat_level_1` will be the flipping point between hurting and not hurting. While `body_temperature` is the temp the body strives towards and thus normalizes to

Reply:
that would make sense

Reply:
Thanks, @farie82! Change made and committed!